docs/src/devel/dev-env-setup.rst

```diff
@@ -178,3 +178,216 @@ e.g.:
+    os_migrate.os_migrate.conversion_host_content : update all packages --------------------------------------- 435.56s
+    os_migrate.os_migrate.import_workloads : transfer volumes to destination ---------------------------------- 101.72s
+    os_migrate.os_migrate.import_workloads : expose source volumes --------------------------------------------- 66.67s
+    os_migrate.os_migrate.conversion_host_content : install content -------------------------------------------- 62.35s
+    os_migrate.os_migrate.import_workloads : transfer volumes to destination ----------------------------------- 58.75s
+    os_migrate.os_migrate.import_workloads : clean up in the source cloud after migration ---------------------- 27.40s
+    os_migrate.os_migrate.import_workloads : expose source volumes --------------------------------------------- 27.30s
+    Create osm_server ------------------------------------------------------------------------------------------ 24.71s
+    create osm_image ------------------------------------------------------------------------------------------- 23.86s
+    os_migrate.os_migrate.export_images : export image blobs --------------------------------------------------- 23.80s
+    os_migrate.os_migrate.import_images : import images -------------------------------------------------------- 23.69s
+    os_migrate.os_migrate.import_workloads : create destination instance --------------------------------------- 23.30s
+    os_migrate.os_migrate.import_workloads : create destination instance --------------------------------------- 21.93s
+    Create osm_server ------------------------------------------------------------------------------------------ 21.61s
+    os_migrate.os_migrate.import_workloads : clean up in the source cloud after migration ---------------------- 21.16s
+    os_migrate.os_migrate.conversion_host : create os_migrate conversion host ---------------------------------- 20.03s
+    Remove osm_server ------------------------------------------------------------------------------------------ 19.01s
+    os_migrate.os_migrate.conversion_host : create os_migrate conversion host ---------------------------------- 18.23s
+    Shutdown osm_server ---------------------------------------------------------------------------------------- 18.14s
+    Shutdown osm_server ---------------------------------------------------------------------------------------- 17.88s
+
+Optional tags to pass to e2e tests
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are a set of tags that can be used to filter which playbooks to run during test.
+
+- test_clean_before
+- test_workload
+- test_image_workload_boot_copy
+- test_image_workload_boot_nocopy
+- test_image_workload_boot_copy_clean
+- test_clean_before
+- test_pre_workload
+
+Optional playbook variable
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+There is also an optional variable `test_clean_conversion_hosts_after` which can be set to `false` if you do not wish
+to cleanup after test is complete.
```

Comment:
"to clean up" -> "to clean up conversion hosts"

(Other cleanup aspects are controlled via the tags above. The conversion hosts cannot be controlled by tags because they're deleted via importing the user-facing playbook which does not have any `test_` tags, and the import_playbook task itself cannot be tagged, so we skip it based on a variable.)

Reply:
Good clarification. Updated